man/sandboxfs.1

```diff
@@ -133,21 +106,18 @@ is not repeated.
 See the
 .Sx Mapping specifications
 subsection for details on how a mapping is specified.
-.It Fl -mem_profile Ar file
-Enables memory profiling and writes a profile to the given
-.Ar file
-on exit.
-.Pp
-This option is incompatible with
-.Fl -listen_address
-because only one profiling mechanism can be enabled at once.
 .It Fl -output Ar path
 Points to the file to which to write confirmations of reconfiguration, or
 .Sq -
 (the default) for stdout.
 See the
 .Sx Reconfigurations
 subsection for details on the contents and behavior of the output file.
+.It Fl -ttl Ar duration
+Specifies how long the kernel is allowed to cache file metadata for.
+The duration is specified as a number of seconds followed by the
```

Comment:
Does it support any other units?

Reply:
Not at the moment. Added "currently" to clarify.